CHANGELOG.md

```diff
@@ -2,17 +2,23 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
-## 9.2.0-alpha.1 - April 21, 2020
+## 9.2.0-beta.1 - April 30, 2020
+
 ### Features
  - The [`SymbolLayer.iconTextFit`](https://docs.mapbox.com/android/api/map-sdk/9.2.0-alpha.1/com/mapbox/mapboxsdk/style/layers/PropertyFactory.html#iconTextFit-com.mapbox.mapboxsdk.style.expressions.Expression-) property now respects the stretch metadata of any nine-part stretchable image passed into the `style.addImage` method. You can define the stretchable area of an image by configuring the `stretchX`, `stretchY`, `content` options on `Style.addImage` to append the requisite [metadata](https://github.com/mapbox/mapbox-gl-js/issues/8917). ([#314](https://github.com/mapbox/mapbox-gl-native-android/pull/314))
  - Added methods to set and get the sort key of features in a `CircleLayer` at runtime. Features with a higher sort key will appear above features with a lower sort key. ([#322](https://github.com/mapbox/mapbox-gl-native-android/pull/322))
  - Added `Source.maxOverscaleFactor` for defining the maximum factor by which a parent tile can be overscaled during rendering. This setting is only needed if a custom source on your map appears blurry or distorted when viewed at new zoom levels. ([#299](https://github.com/mapbox/mapbox-gl-native-android/pull/299))
+ - Added methods to set and get `volatile` in `Source`. Set `volatile` to false will store tiles in local cacje. ([#355](https://github.com/mapbox/mapbox-gl-native-android/pull/355))
+ - Added methods to set and get `minimumTileUpdateInterval` in `Source`. `minimumTileUpdateInterval` is used to throttle the tile update network requests([#355](https://github.com/mapbox/mapbox-gl-native-android/pull/355))
```

Comment:
Definitely understand how you got this changelog entry from the GL Native changelog 😅 I think this language is somewhat misleading, as I pointed out [here](https://github.com/mapbox/mapbox-gl-native/pull/16416#discussion_r412719421). The main point of this feature is to allow a customer to override the server-side expiration (time-to-live or ttl) with a longer client-side ttl.